codexctl/sync.py

```
@@ -216,0 +1,218 @@
+                self.logger.error("Request timed out!")
+
+            logging.debug(f"Result headers: {result.headers}")
+            if "application/json" in result.headers["Content-Type"]:
+                return result.json()
+            return result.content
+
+        except Exception:
+            return None
+
+    def __get_documents_recursive(
+        self, folderId="", currentLocation="", currentDocuments=[]
```

Ruff (0.8.0)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)